src/elm/Vela.elm

```diff
@@ -388,22 +393,42 @@ type alias RepoModel =
 type alias BuildModel =
     { buildNumber : BuildNumber
     , build : WebData Build
-    , steps : WebData Steps
+    , steps : StepsModel
```

Comment:
the reason for moving steps beneath its own model will be much more obvious in the next PR when I add the rest of the services code.

this lets us preserve line focus and follow state for multiple resources (steps and services)!!